storm-server/src/main/java/org/apache/storm/daemon/metrics/MetricsUtils.java

```diff
@@ -58,10 +61,11 @@ private static PreparableReporter getPreparableReporter(String clazz) {
         return reporter;
     }
 
-    public static File getCsvLogDir(Map<String, Object> topoConf) {
-        String csvMetricsLogDirectory = ObjectReader.getString(topoConf.get(DaemonConfig.STORM_DAEMON_METRICS_REPORTER_CSV_LOG_DIR), null);
+    public static File getCsvLogDir(Map<String, Object> daemonConf) {
```

Comment:
Unrelated to this PR, but it seems odd this is a metrics util thing rather than belonging to the CSV reporter class.

Reply:
Yeah, but for compatibility, I can't simply move it. If any topology invokes it directly, and if we move it and upgrade the cluster, this old topology will has ClassNotFoundException.